components/user-avatar/src/text-avatar.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export const getInitials = (name) =>
+    name
```

Comment:
🤔 Good point. I think first and last word is probably going to produce the desired outcome in more cases. But actually it would probably be even more accurate to not feed a full name into this component, but provide a first and last name instead. And then I guess we could take the first letter of the first word of each field....

For my quirky language, it would actually be even better to take the first capitalised word. For example, my last name is "de Graaf" but "G" should be the letter used in my avatar.

At the very least this comment highlights that it makes a lot of sense to extract this component. We don't want the same users to have different avatars in different places in the system.

Reply:
I've adjusted the implementation in this component to use the first and last word in the name as @edoardo suggested.